Src/Support/Google.Apis/Services/BaseClientService.cs

```diff
@@ -229,12 +229,6 @@ public void SetRequestSerailizedContent(HttpRequestMessage request, object body)
         /// <inheritdoc/>
         public virtual string SerializeObject(object obj)
```

Comment:
Make this an expression-bodied method now it's so simple?

Reply:
Done :)